### ISSUE TITLE
BanManager support

### ISSUE THREAD

**Issue by Nety01** [OopsieWoopsie/BanAnnouncer#6]
Could you please add support for this plugin? https://www.spigotmc.org/resources/banmanager.21927/

**Comment by OopsieWoopsie** [comment]
Sure, but I will take this as low priority because I have other things to do, but due I haven't updated BA. for a long time I think I will do this asap when I have free time.

**Comment by Blocksnmore** [comment]
I would recommend using Advancedban https://www.spigotmc.org/resources/advancedban.8695/ over ban manager since it has more features mysql and bungee support along with a web terminal to manage bans if you set it up and host it. Thats just my opinion tho

**Comment by Nety01** [comment]
The dev of BanManager added DiscordSRV support.

**Comment by OopsieWoopsie** [comment]
I have almost "abandoned" this when I saw that that BanManager provides kicks but it doesn't have a KickEvent and also it has the events for Un-Action (like PlayerUnban) but the UnwarnEvent is missing (BA doesn't "supports" Un-Actions but it was going to be released with the BanManager support), also the IpBannedEvent only provides the IP address in Long format, I can't get the player name using the data provided by that event (idc the format of the IP, it can be easily converted).

If I release support to BanManager it will just make BanAnnouncer incomplete for the users that uses that plugin.

I didn't give up, I just took away all the priority I gave to it.
